apps/examples/wc-next10/pages/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Head from 'next/head';
+import dynamic from 'next/dynamic'
+
+const PButton = dynamic(() => import('../ButtonLogic'), { ssr: false });
+
+export default function Home() {
+    return (
+        <>
+            <Head>
+                <title>Create Next App</title>
+                <meta name="viewport" content="width=device-width, initial-scale=1" />
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
```

Comment:
I believe that in nextjs, you can specify the head globally for all pages in _document.jsx as this will only attach the title meta to the index page "/" only. i suggest we move it to `_document.jsx` just in case we add a new page in the fiture.